src/renderer/components/wallet/assets/AssetsTableCollapsable.stories.tsx

```diff
@@ -30,33 +29,6 @@ const assetHandler = ({ asset, walletType, walletAddress }: SelectedWalletAsset)
   console.log('assetHandler params ', assetToString(asset), walletType, walletAddress)
 const disableRefresh = false
 const balances: Partial<Record<EnabledChain, ChainBalances>> = {
-  [BNBChain]: [
```

Comment:
why remove completely? swap with another asset

Reply:
Just removing the BNB chain as an option.

Reply:
Maya added in